guides/common/modules/ref_dhcp-isc-settings.adoc

```diff
@@ -0,0 +1,43 @@
+[id="DHCP_ISC_Settings_{context}"]
+= dhcp_isc Settings
+
+The _dhcp_isc_ provider uses a combination of the ISC DHCP server OMAPI management interface and parsing of configuration and lease files.
+This requires it to be run either on the same host as the DHCP server or to have network filesystem access to these files.
+You can define the following settings in `dhcp_isc.yml`:
+
+ifdef::foreman-el,katello[]
+.Configuring the path to the _config_ and _leases_ files on Enterprise Linux
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+:config: /etc/dhcp/dhcpd.conf
+:leases: /var/lib/dhcpd/dhcpd.leases
+----
+endif::[]
+
+ifdef::foreman-deb[]
+.Configuring the path to the _config_ and _leases_ files on Debian or Ubuntu:
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+:dhcp_config: /etc/dhcp3/dhcpd.conf
+:dhcp_leases: /var/lib/dhcp3/dhcpd.leases
+----
+endif::[]
+
+.Securing the DHCP server with an _omapi_key_
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+:key_name: _My_OMAPI_Key_
+:key_secret: _My_Key_Secret_
+----
+
+.Setting a port on which the DHCP server listens
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+:omapi_port: _My_DHCP_Server_Port_  # default: 7911
+----
+
+.Setting the host on which the DHCP server runs on
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+:server: _My_DHCP_Server_FQDN_
```

Comment:
`server` is in `dhcp.yml`, not `dhcp_isc.yml`.

Reply:
Thanks, applied!